src/System.Private.CoreLib/shared/System/Number.Dragon4.cs

```diff
@@ -277,7 +277,7 @@ private static unsafe uint Dragon4(ulong mantissa, int exponent, uint mantissaHi
             }
 
             // Output the exponent of the first digit we will print
-            decimalExponent = digitExponent - 1;
+            decimalExponent = --digitExponent;
```

Comment:
We generally seem to avoid prefix notation and instead write this as two lines for clarity.

Reply:
I think that the following can be just as confusing
```csharp
digitExponent--;
decimalExponent = digitExponent;
```

Reply:
Up to you, unless we have some policy somewhere..

Reply:
I don't see one in our project docs, and I see a couple other places (such as in StringBuilder, StreamReader, various parsing code, and various bits of native code) that use the prefix notation as well, so I'm probably going to keep it as is.